.github/workflows/build-rapids.yml

```diff
@@ -0,0 +1,65 @@
+name: Build all RAPIDS repositories
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_call:
+
+jobs:
+  check-event:
+    name: Check GH Event
+    runs-on: ubuntu-latest
+    outputs:
+      ok: ${{ steps.check_gh_event.outputs.ok }}
+    steps:
+      - id: check_gh_event
+        name: Check GH Event
+        shell: bash
+        run: |
+          [[ '${{ github.event_name }}' == 'push' && '${{ github.repository }}' == 'NVIDIA/cccl' ]] || \
+          [[ '${{ github.event_name }}' == 'schedule' && '${{ github.repository }}' == 'NVIDIA/cccl' ]] || \
+          [[ '${{ github.event_name }}' == 'pull_request' && '${{ github.repository }}' != 'NVIDIA/cccl' ]] \
+          && echo "ok=true"  | tee -a $GITHUB_OUTPUT \
+          || echo "ok=false" | tee -a $GITHUB_OUTPUT;
+
+  build-rapids:
+    name: Build all RAPIDS repos
+    if: needs.check-event.outputs.ok == 'true'
+    needs: check-event
+    secrets: inherit
+    # TODO: this is temporary
+    uses: ./.github/workflows/build-in-devcontainer.yml
+    # uses: rapidsai/shared-workflows/.github/workflows/build-in-devcontainer.yaml@branch-24.06
+    permissions:
+      actions: read
+      packages: read
+      id-token: write
+      contents: read
+      pull-requests: read
+    with:
+      arch: '["amd64"]'
+      cuda: '["12.2"]'
+      node_type: cpu32
+      build_command: |
+        # Remove cumlprims_mg and cugraph-ops from /opt/rapids-build-utils/manifest.yaml
+        sudo yq -i '.repos |= filter(.name != "cumlprims_mg" and .name != "cugraph-ops")' /opt/rapids-build-utils/manifest.yaml;
+        # Regenerate builds scripts after modifying /opt/rapids-build-utils/manifest.yaml
+        rapids-generate-scripts;
+        # Clone all the RAPIDS repos
+        clone-all -j$(nproc) -v -q --clone-upstream --single-branch --shallow-submodules;
```

Comment:
Instead of manually removing cumlprims_mg and cugraph-ops, could you just clone the subset of repos that you want?

Reply:
That's effectively what I'm doing here. By removing these, I can call `{clone,build}-all` and not have to list every library individually. We probably do want to build those too but that requires adding the repo deploy keys as secrets in CCCL, and I don't have access to do that.

Reply:
All of the ones listed [here](https://github.com/rapidsai/devcontainers/blob/7013588ebeefaba105849098af07d3ef7208eaec/features/src/rapids-build-utils/opt/rapids-build-utils/manifest.yaml), minus `cumlprims_mg` and `cugraph-ops`.